### PR TITLE
PT-1955

### PR DESCRIPTION
**Pull request by CVirus**
- Add initial ahoy and ci scaffolds

### CODE REVIEW

**Comment by CVirus** [comment]
Don't review dashboard.json yet.

File: config.yml [hunk 1]
```diff
@@ -1,10 +1,31 @@
...
 instances:
-  - instance: rds-aurora1
-    region: us-east-1
-  - instance: rds-aurora57
-    region: us-east-1
-  - instance: rds-mysql56
-    region: us-east-1
-  - instance: rds-mysql57
-    region: us-east-1
+  - instance: auth-service-db000-live
```
Comment (rafaeljesus) [inline config.yml]: is there any option for fetching all the instances w/out the need to adding to this file?

Comment (CVirus) [reply config.yml]: Done .. please re-review

File: entry.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
```
Comment (rafaeljesus) [inline entry.py]: hm I am not really sure about having py and go. Why not do it in go?

Comment (rafaeljesus) [reply entry.py]: If the script is too long for be managed by `sh`, we usually write it in go

Comment (CVirus) [reply entry.py]: Simply because I don't know go.. yet :D

**Comment by rafaeljesus** [comment]
:shipit:

**Pull request merged**